nautobot/extras/api/serializers.py

```diff
@@ -918,7 +918,7 @@ def validate(self, data):
 class JobInputSerializer(serializers.Serializer):
     data = serializers.JSONField(required=False, default=dict)
     commit = serializers.BooleanField(required=False, default=None)
-    schedule = NestedScheduledJobSerializer(required=False)
+    scheduled_job = NestedScheduledJobSerializer(required=False)
```

Comment:
This change probably needs to be documented somewhere in the upgrading guide (it's not a model serializer per se so it doesn't fit in the current tables, though).

This will also need to be updated in `nautobot/docs/additional-features/job-scheduling-and-approvals.md` and probably `nautobot/docs/additional-features/jobs.md` as well.

It seems like a similar change to the `_schedule` fields in `JobMultiPartInputSerializer` should also be made and reflected in `nautobot.extras.api.views._run_job`?

Reply:
FYI I'm going to revert this particular change and the associated changes to `_run_job` and the tests in #3368. This is because this isn't actually a nested `ScheduledJob` object reference, but rather just a set of parameters to the `run_job` function that we represent in a fashion "similar to" a `ScheduledJob`.